bang/forms.py

```diff
@@ -600,6 +600,9 @@ def __init__(self, *args, **kwargs):
         if 'c_versions' in self.fields:
             del(self.fields['c_versions'])
 
+    def clean_japanese_name(self):
+        return self.cleaned_data.get('japanese_name')[:-3] if self.cleaned_data.get('japanese_name').endswith(u'ガチャ') else self.cleaned_data.get('japanese_name')
```

Comment:
it would be cleaner to save the japanese_name in a variable so you don't have to .get() it every time.